### PR TITLE
update prose for website

### PR DESCRIPTION
**Pull request by isabelizimm**
Updating prose for website to make more beginner-friendly SRE landing page

### CODE REVIEW

**Comment by schwesig** [comment]
3 Links are not working correctly (old cluster ?!)

1. http://prometheus-portal-opf-monitoring.apps.zero.massopen.cloud/graph/
2. https://grafana-route-opf-monitoring.apps.zero.massopen.cloud/
3. http://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/hub/login

File: README.md [hunk 1]
```diff
@@ -6,37 +6,12 @@ This repository contains all the SRE (Site Reliability Engineering) principles a
...
 
-## SLI and SLOs
-
-For each of the Operate First managed services, we define Service Level Indicators (SLI) and Service Level Objectives (SLO) to help us monitor the availability of the service and improve its reliability.
-
-The `sli-slo` folder [here](https://github.com/operate-first/SRE/tree/master/sli-slo) lists out the SLI/SLO for each of the applications in separate markdown files. For eg, you can find the SLI/SLO defined for the `JupyterHub` application [here](https://github.com/operate-first/SRE/blob/master/sli-slo/jupyterhub.md).
-
-## Operational Tools
-
-* [Prometheus](http://prometheus-portal-opf-monitoring.apps.zero.massopen.cloud/graph) - Monitoring tool capturing time series metrics for available Operate First services
```
Comment (schwesig) [inline README.md:24]: Link is not working

File: sli-slo/tools.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+This page highlights the operational tools and services monitored through Operate First.
+
+## Operational Tools
+
+* [Prometheus](http://prometheus-portal-opf-monitoring.apps.zero.massopen.cloud/graph) - Monitoring tool capturing time series metrics for available Operate First services
```
Comment (schwesig) [inline sli-slo/tools.md]: Link is not working: old cluster?
Change Link.

File: README.md [hunk 3]
```diff
@@ -6,37 +6,12 @@ This repository contains all the SRE (Site Reliability Engineering) principles a
...
-* [Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) - Alertmanager for handling alerts and routing them to the appropriate receiver integration such as Email, Slack, PagerDuty etc
-* [Grafana](https://grafana-route-opf-monitoring.apps.zero.massopen.cloud/) - Visualization tool for creating monitoring dashboards
```
Comment (schwesig) [inline README.md:26]: Link is not working: old cluster?
Change Link.

File: sli-slo/tools.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+* [Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) - Alertmanager for handling alerts and routing them to the appropriate receiver integration such as Email, Slack, PagerDuty etc
+* [Grafana](https://grafana-route-opf-monitoring.apps.zero.massopen.cloud/) - Visualization tool for creating monitoring dashboards
```
Comment (schwesig) [inline sli-slo/tools.md]: Link is not working: old cluster?
Change Link.

File: README.md [hunk 5]
```diff
@@ -6,37 +6,12 @@ This repository contains all the SRE (Site Reliability Engineering) principles a
...
 SRE is a software engineering approach to manage operations for systems, applications and services. We use software as a tool to manage systems, solve problems, and automate operations tasks.
 
-## Services Monitored
+## Get started
 
-We have [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC [(Mass Open Cloud)](https://massopen.cloud/) cluster. Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers.
+If you'd like to learn SRE practices, but aren't sure where to start, let us help!
 
-The components we currently have available are:
-* [JupyterHub](https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/hub/login)
```
Comment (schwesig) [inline README.md:14]: Link is not working: old cluster?
Change Link.

File: sli-slo/tools.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+* [GitHub Receiver](https://github.com/m-lab/alertmanager-github-receiver) - Incident reporting tool for handling outages/incidents
+
+## Services Monitored
+
+We have [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC [(Mass Open Cloud)](https://massopen.cloud/) cluster. Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers.
+
+The components we currently have available are:
+* [JupyterHub](https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/hub/login)
```
Comment (schwesig) [inline sli-slo/tools.md]: Link is not working: old cluster?
Change Link.

File: README.md [hunk 7]
```diff
@@ -6,37 +6,12 @@ This repository contains all the SRE (Site Reliability Engineering) principles a
...
+1. Follow this link to find beginner-friendly issues [beginner friendly issues](https://github.com/operate-first/support/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
```
Comment (hemajv) [inline README.md]: "beginner friendly issues" mentioned twice, remove and link to just one?

File: sli-slo/tools.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+* [Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) - Alertmanager for handling alerts and routing them to the appropriate receiver integration such as Email, Slack, PagerDuty etc
+* [Grafana](https://grafana.com/) - Visualization tool for creating monitoring dashboards
+* [GitHub Receiver](https://github.com/m-lab/alertmanager-github-receiver) - Incident reporting tool for handling outages/incidents
+
+## Services Monitored
+
+We have [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC [(Mass Open Cloud)](https://massopen.cloud/) cluster. Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers.
+
+The components we currently have available are:
+* [JupyterHub](https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/login)
```
Comment (hemajv) [inline sli-slo/tools.md]: Do we want to add a short description to what JupyterHub is? and also maybe point to some resource on the Operate First website?

File: README.md [hunk 9]
```diff
@@ -6,37 +6,12 @@ This repository contains all the SRE (Site Reliability Engineering) principles a
...
-
-The [`incident-management`](https://github.com/operate-first/SRE/tree/master/incident-management) folder contains more information on how to setup GitHub alertmanager receiver and configure Prometheus alerts.
-
-## Runbooks
-
-For each of the services being monitored, we aim to define runbooks for providing information on how to administer, debug and effectively troubleshoot common problems of a service.
-
-These runbooks (when created), will be defined in the [`runbooks`](https://github.com/operate-first/SRE/tree/master/runbooks)
-folder.
+To learn more, check out the [incident management procedures](./incident-management/incident-management-procedure.md), [GitHub receiver setup](./incident-management/github-receiver-setup.md), learn to [configure Prometheus alerts](./incident-management/configure-prometheus-alerts.md), or browse the [GitHub repo](https://github.com/operate-first/support).
```
Comment (hemajv) [inline README.md]: Link all the resources to the Operate First website instead?

- https://www.operate-first.cloud/operations/sre/incident-management/incident-management-procedure.md
- https://www.operate-first.cloud/operations/sre/incident-management/github-receiver-setup.md
- https://www.operate-first.cloud/operations/sre/incident-management/configure-prometheus-alerts.md

File: README.md [hunk 10]
```diff
@@ -6,37 +6,12 @@ This repository contains all the SRE (Site Reliability Engineering) principles a
 
 SRE is a software engineering approach to manage operations for systems, applications and services. We use software as a tool to manage systems, solve problems, and automate operations tasks.
 
-## Services Monitored
+## Get started
 
-We have [Open Data Hub](https://opendatahub.io/) applications deployed and running in a MOC [(Mass Open Cloud)](https://massopen.cloud/) cluster. Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers.
+If you'd like to learn SRE practices, but aren't sure where to start, let us help!
```
Comment (hemajv) [inline README.md]: maybe we can update it to "If you'd like to learn and get hands on experience with SRE practices, but aren't sure where or how to start, let us help!"?

**Review by hemajv — CHANGES_REQUESTED** [review]

**Review by schwesig — APPROVED** [review]

**Review by hemajv — APPROVED** [review]

**Pull request merged**